src/SDK/Common/Export/InMemoryStorageManager.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common\Export;
+
+use ArrayObject;
+
+class InMemoryStorageManager
+{
+    private static ArrayObject $spans;
+    private static ArrayObject $metrics;
+    private static ArrayObject $logs;
+
+    public static function getStorageForMetrics(): ArrayObject
```

Comment:
Stylistic change: I think these method names could be shortened to `metrics`, `logs`, and `spans`